arangodb-net-standard/ViewApi/ViewsApiClient.cs

```diff
@@ -187,13 +196,18 @@ public ViewApiClient(IApiClientTransport transport, IApiClientSerialization seri
         /// </summary>
         /// <param name="viewName">The name of the view.</param>
         /// <param name="body">The body of the request containing required properties.</param>
+        /// <param name="ignoreNullValuesOnSerialization">Indicates whether to ignore null values during serialization.</param>        
```

Comment:
Minor: Should we add a bit more information about how it can be used? e.g.
```c#
/// <param name="ignoreNullValuesOnSerialization">
/// Indicates whether to ignore null values during serialization.
/// Use this parameter together with <see cref="LinkProperties.IncludeAllFields"/> set to false to control if fields with null values are included.
/// </param>
```
Not a blocker.

Reply:
Added.